.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: cpp
 compiler: gcc
-cache: ccache
+cache:
+  ccache: true
+  directories:
+    - /var/cache/apt/archives
 sudo: required
 dist: trusty
 install:
-  - ./utils/travis-dependencies.sh
+  - ./utils/travis/dependencies.sh
 script:
-  - cd utils
-  - source /opt/qt*/bin/qt*-env.sh
-  - ./travis-build.sh
+  - . utils/travis/common.sh
+  - env_setup
```

Comment:
This will not work if the Qt 5.9 backports are not installed, like in the future for the click build

Reply:
The function checks if qt exsists!
https://github.com/KaidanIM/Kaidan/pull/208/files#diff-1c37644e56f5e80d16112e49505e1d1fR61

Reply:
Oh sorry, missed that :)